packages/portal-contracts/src/deploy/deploy.s.sol

```diff
@@ -1,22 +1,23 @@
 // SPDX-License-Identifier: MIT
 pragma solidity >=0.8.0;
 
-import "forge-std/Test.sol";
+import "forge-std/Script.sol";
 import "btcmirror/BtcMirror.sol";
 import "btcmirror/BtcTxVerifier.sol";
 import "../Portal.sol";
 
-contract PortalDeploy is Test {
-    function run(bool mainnet, IERC20 token, BtcMirror existingMirror)
-        public
+contract PortalDeploy is Script {
+    function run()
+       external 
```

Comment:
to pass args to the script:
```
forge script (forge args) -- (script args)
```

we can hardcode `token` to WBTC, but I think mainnet & existingMirror are still good args bc we don't want to redeploy BtcMirror each time